docs/support/coverage.md

```diff
@@ -0,0 +1,27 @@
+# Coverage 
+
+To get the most out of Steamship's cloud platform you'll be using adapters. Steamship adapters serve as drop-in replacement for LangChain's modules. Here's an overview of the status of our adapters and their coverage:
+
+
+## Modules 
+
+* LLMs
+  * ✅ LLM Models
+  * ✅ LLM Caching 
+  * ❌ LLM Serialisation
+  * ✅ Token Usage Tracking 
+* Prompt Templates ✅
+* Chains ✅
+* Agents 
+  * ✅ SERPAPI
+* Memory
+  * ✅ Complete Memory 
+  * ✅ Windowed Memory 
+
+
+## Use Cases
+
+* Chatbot 
+  * ✅ ChatGPT
```

Comment:
let's add `Question Answering`.  Either here, or in a follow-on PR.

Also, wdyt about `Summarization` ?

Reply:
Haven't tried it. I think a follow-on PR is useful here since we probably want to reflect support int he docs too.